src/ui-kit/experimental/hierarchical/hierarchical-tree-header/hierarchical-tree-header.component.html

```diff
@@ -1,3 +1,18 @@
-<p>
-  hierarchical-tree-header works!
-</p>
+<div>
+  <!-- View Selector-->
+  <div>
```

Comment:
@yerramshilpa Let's ask Christy/Lorrin/Kapil if they still want to display the view selector now or later.